src/Auth0.ManagementApi/IManagementConnection.cs

```diff
@@ -26,7 +27,7 @@ public interface IManagementConnection
         /// <param name="headers">Dictionary containing additional headers that may override the defaults.</param>
         /// <param name="converters">Optional array of <see cref="JsonConverter"/>s used to deserialize the resulting <typeparamref name="T"/>.</param>
         /// <returns><see cref="Task"/> representing the async operation containing response body as <typeparamref name="T"/>.</returns>
-        Task<T> GetAsync<T>(Uri uri, IDictionary<string, string> headers, JsonConverter[] converters = null);
+        Task<T> GetAsync<T>(Uri uri, IDictionary<string, string> headers, JsonConverter[] converters = null, CancellationToken token = default);
```

Comment:
I know the PR is still draft but I did want to call out this ... 

This introduces a breaking change, anyone that is implementing our `IManagementConnection` interface will need to update its implementation, which I would like to avoid if possible.

Reply:
Did you have a design in mind to avoid the break? I assume the management connection interface primarily exists for testing reasons and that might be less of a problem than a normal API break.

Reply:
> I assume the management connection interface primarily exists for testing reasons and that might be less of a problem than a normal API break.

I believe it can be used by anyone who wants to take control over the HttpClient, e.g. for configuring timeouts or anything.

> Did you have a design in mind to avoid the break?

I can't really think of anything that both makes sense and doesn't break users. So I guess we will need to stick to the current implementation and roll with a small breaking change.

I believe we follow SemVer best effort, meaning that if we believe a small breaking change can make sense without bumping the major version, we should do so.

Reply:
> I believe we follow SemVer best effort, meaning that if we believe a small breaking change can make sense without bumping the major version, we should do so. 

Seems reasonable to me. I'll continue work on this PR in the next few days 👍